sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/pipeline/pipeline_job.py

```diff
@@ -470,6 +470,15 @@ def _to_rest_object(self) -> JobBase:
         # This will be resolved in job_operations _resolve_arm_id_or_upload_dependencies.
         component_id = self.component if isinstance(self.component, str) else self.component.id
 
+        # MFE not support pass None or empty input value. Remove the empty inputs and parameters in jobs.
```

Comment:
created a task to server to accept binding to empty inputs

Reply:
Created a task to backend